js/delta.js

```diff
@@ -185,36 +185,76 @@ module.exports = class delta extends Exchange {
 
     async fetchTime (params = {}) {
         const response = await this.publicGetSettings (params);
-        //
-        //     {
-        //         "result":{
-        //             "server_time":1605472733766141,
-        //             "deto_referral_mining_daily_reward":"25000",
-        //             "deto_total_reward_pool":"100000000",
-        //             "deto_trade_mining_daily_reward":"75000",
-        //             "kyc_deposit_limit":"20",
-        //             "kyc_withdrawal_limit":"2",
-        //             "under_maintenance":"false"
-        //         },
-        //         "success":true
-        //     }
-        //
+        // full response sample under `fetchStatus`
         const result = this.safeValue (response, 'result', {});
         return this.safeIntegerProduct (result, 'server_time', 0.001);
     }
 
     async fetchStatus (params = {}) {
         const response = await this.publicGetSettings (params);
```

Comment:
i don't think we should dupe the fetchTime endppoint to here

Reply:
I remind that one endpoint can be used by different unified methods across different exchanges/implementations.  for this specific `fetchStatus` too, there has also been other cases similar to that iirc, to serve that simple purpose to fetch status.
idk as you think